src/styles/blocks/modifiers.scss

```diff
@@ -0,0 +1,3 @@
+.stars--4 .stars__item:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
```

Comment:
It's good that you've used BEM for your class naming, however, it seems that you have a code style issue here. According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) you should always leave a space before the opening curly brace `{`. Please adjust your code accordingly.